tests/nvidiagpu/gpudeploy/gpu_suite_test.go

```diff
@@ -0,0 +1,27 @@
+package nvidiagpu
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/rh-ecosystem-edge/nvidia-ci/tests/internal/reporter"
+
+	"github.com/rh-ecosystem-edge/nvidia-ci/tests/nvidiagpu/gpudeploy/internal/tsparams"
+	_ "github.com/rh-ecosystem-edge/nvidia-ci/testsl/nvidiagpu/gpudeploy/tests"
```

Comment:
```suggestion
	_ "github.com/rh-ecosystem-edge/nvidia-ci/tests/nvidiagpu/gpudeploy/tests"
```

Reply:
Good catch, thank you, will fix in next commit.

Reply:
Fixed, good catch, thank you @mresvanis !